### DISCUSSION TITLE
navigate to html packed in resource in WPF

### DISCUSSION THREAD

**Discussion by issam1975** [MicrosoftEdge/WebView2Feedback#3898] [Q&A]
hi

is there a way to navigate to an html/javascript content embedded as resource in a wpf application ?

thanks .

**Comment by champnic** [comment]
Hey @issam1975 - there isn't a "built-in" way to do this. Instead your app will need to:
1. Listen to NavigationStarting to your resource files
2. Cancel that navigation
3. Load the HTML resource from the native side
4. NavigateToString(Html string) OR Save the html string to a file and navigate to that file

If you search these GitHub issues I believe there are other folks who have done similar things.
